timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py

```diff
@@ -317,7 +317,7 @@ def on_train_begin(self, args, state, control, **kwargs):
     def on_step_end(self, args, state, control, **kwargs):
         elapsed_time = time.monotonic() - self.start_time
         if elapsed_time > self.time_limit:
-            logger.info("\tStopping fine-tuning since time_limit is reached")
+            logger.log(15, "Stopping fine-tuning since time_limit is reached")
```

Comment:
Why not `info`?

Reply:
Info (default) usually should only report the val_loss, fit time and predict time. Information about early stopping seems much more detailed, so we reserve it for verbosity=3

Reply:
Also, stopping training when the time_limit is reached is the default behavior for most models (e.g., all GluonTS models), and we don't usually log that.